### ISSUE TITLE
Specifically disable cache when doing requests through fetch

### ISSUE THREAD

**Issue by creesch** [toolbox-team/reddit-moderator-toolbox#413]
I ran into this today

![image](https://user-images.githubusercontent.com/2502190/97981296-574e0280-1dd2-11eb-8ef5-7770fdecf371.png)

It is probably best to specifically set fetch to use `no-store` or `no-cache`. I am personally leaning towards `no-store` as I don't really trust reddit to properly respond to a conditional request. 

https://developer.mozilla.org/en-US/docs/Web/API/Request/cache

**Comment by creesch** [comment]
Seems like when the API is working properly the response header is giving a `no-store`, which is missing with a 503 error. 

A bit of a challenge I came across though when trying to experiment with it is that the `cache` property in the request doesn't appear to do anything when you look at the request in the browser. At least, it doesn't appear to change the headers as that remain the same. 

I am not sure if I am misunderstanding the property wrong and have the wrong expectations or that it doesn't work properly due to something else. 

[I did come across this](https://stackoverflow.com/questions/29246444/fetch-how-do-you-make-a-non-cached-request) with different methods of achieving no caching including changing the header itself but frankly I rather make use of the api methods if at all possible.
